utils/neuralmagic/utils.py

```diff
@@ -81,20 +96,27 @@ def load_sparsified_model(
     # Load checkpoint if not yet loaded
     ckpt = ckpt if isinstance(ckpt, dict) else torch.load(ckpt, map_location=device)
 
-    # Construct randomly initialized model model and apply sparse structure modifiers
-    model = Yolov5Model(ckpt.get("yaml"))
-    model = update_model_bottlenecks(model).to(device)
-    checkpoint_manager = ScheduledModifierManager.from_yaml(ckpt["checkpoint_recipe"])
-    checkpoint_manager.apply_structure(
-        model, ckpt["epoch"] + ALMOST_ONE if ckpt["epoch"] >= 0 else float("inf")
-    )
+    if isinstance(ckpt["model"], torch.nn.Module):
```

Comment:
for cases when "model" is not in checkpoint?
```suggestion
    if isinstance(ckpt.get("model"), torch.nn.Module):
```

Reply:
Usually a good idea, but I'd rather fail here, as ckpt should always have a `"model"` key and one without is truly broken